analyzers/tests/SonarAnalyzer.UnitTest/TestCases/UseReturnStatement.vb

```diff
@@ -0,0 +1,103 @@
+﻿Public Class ImplicitReturnStatementsAreNoncompliant
+    Public Function AssignedReturnValueOnly() As Integer
+        AssignedReturnValueOnly = 42 ' Noncompliant ^9#23
+    End Function
+	
+	Public Function AssignementStatements() As Integer
```

Comment:
With the last commit, tab character appeared in this file. Can you please fix those?
In the VS, I also see tab character on line 5. It's not visible on GitHub UI.

I use this [Trailing Whitespace Visualizer](https://marketplace.visualstudio.com/items?itemName=MadsKristensen.TrailingWhitespaceVisualizer) extension to see whitespace, it's useful and nonintrusive.

Reply:
Because the IDE 'fixes' the uppercase mismatch automatically, I edited the test case file with Notepad++, hence the tabs.